docs/templating-functions.md

```diff
@@ -0,0 +1,6 @@
+# Templating functions
+
+On top of golangs normal templating we provide some functions to make writitng references easier.
+The (sprig functions)[http://masterminds.github.io/sprig/] are included along with some other functions that can be found in the (subpackages pkg.go.dev page)[https://pkg.go.dev/github.com/openshift/kube-compare/pkg/funcmap#pkg-functions]
+
+If you want to custom functions you can define them as a templates and include them a paths under `templateFunctionFiles` at the root of your reference `metadata.yml`
```

Comment:
Maybe an example or two (sprig && custom) would be nice!

Reply:
Not sure what you mean here? 
Do you mean you want me to list a few or show a template with them in?

Reply:
Yep that's it. Show a template with them...so like as a user I would like tips on how to incorporate spring/template  funcs in my existing references! Also not sure what `templateFunctionFiles` is...a custom dir name? (but including an example should clear it up)

btw metadata.yml should be metadata.`yaml`? The file ext is somewhat hardcoded right?